src/endpoints/__tests__/createTunewsLanguageEndpoint.spec.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import LanguageModel from '../../models/LanguageModel'
+import createTunewsLanguagesEndpoint from '../createTunewsLanguagesEndpoint'
+
+describe('tunews language', () => {
+  const baseUrl = 'https://cms-test.integreat-app.de'
+  const tunewsElement = createTunewsLanguagesEndpoint(baseUrl)
+
+  const languagesJson = [
```

Comment:
Maybe add a rtl language here.

Reply:
I have just seen that the direction is not yet added to the tunews model. https://github.com/Integreat/tunews/blob/master/doc/api.md
Therefore I'll remove it again from tunews